src/rp2_common/pico_bootrom/include/pico/bootrom.h

```diff
@@ -173,7 +173,7 @@ __force_inline static void *rom_hword_as_ptr(uint16_t rom_address) {
 
 #ifdef __riscv
 static __force_inline bool rom_size_is_64k(void) {
-#ifdef RASPBERRYPI_AMETHYST_FPGA
+#if RASPBERRYPI_FPGA
```

Comment:
Should `src/rp2_common/pico_crt0/crt0_riscv.S` also be updated to use `RASPBERRYPI_FPGA` instead of `RASPBERRYPI_AMETHYST_FPGA` ? :shrug:

Reply:
i have reverted that part of the change as it is actually subtly broken I now realize